oshi-core/src/main/java/oshi/software/os/unix/freebsd/FreeBsdOperatingSystem.java

```diff
@@ -127,14 +127,14 @@ public List<OSProcess> queryAllProcesses() {
     public List<OSProcess> queryChildProcesses(int parentPid) {
         List<OSProcess> allProcs = queryAllProcesses();
         Set<Integer> descendantPids = getChildrenOrDescendants(allProcs, parentPid, false);
-        return allProcs.stream().filter(p -> descendantPids.contains(p.getProcessID())).collect(Collectors.toList());
+        return allProcs.stream().parallel().filter(p -> descendantPids.contains(p.getProcessID())).collect(Collectors.toList());
```

Comment:
very little benefit if any (repeating myself, all of these are parallelizing set.contains.... which is O(1) fast)